engine/userguide/networking/index.md

```diff
@@ -394,19 +394,24 @@ by creating an `overlay` network.
 
 ### The `docker_gwbridge` network
 
-This is a local bridge network which is automatically created by docker.
-When a container is being connected to a network which cannot provide external connectivity
-(and none of the networks to which the container is already connected can provide external
-connectivity), docker also automatically connects the container to the `docker_gwbridge` network.
-So that the container can reach and can be reached from the internet. 
+The `docker_gwbrodge` is a local bridge network which is automatically created by Docker
+when you initialize or join a swarm, or when none of a container's networks can provide
```

Comment:
I think we should terminate the sentence here: `[...] by Docker. When you [...]`